test/e2e/cron_test.go

```diff
@@ -19,22 +17,50 @@ import (
 	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
 	"github.com/argoproj/argo/test/e2e/fixtures"
 	"github.com/argoproj/argo/workflow/common"
+	"github.com/argoproj/pkg/humanize"
 )
 
 type CronSuite struct {
 	fixtures.E2ESuite
 }
 
 func (s *CronSuite) SetupSuite() {
-	if os.Getenv("SKIP_CRON_SUITE") == "true" {
-		s.T().SkipNow()
-	}
 	s.E2ESuite.SetupSuite()
+	// Since tests run in parallel, delete all cron resources before the test suite is run
+	s.E2ESuite.DeleteResources(fixtures.LabelCron)
```

Comment:
how do you deal with offloaded data which is just nuked? perhaps we don't need to nuke argo_workflows/argo_archived_workflows?

Reply:
These tests are agnostic to offloaded data